CatProducer/python/secondaryVertexProducer_cfi.py

```diff
@@ -1,9 +1,9 @@
 import FWCore.ParameterSet.Config as cms
 
 catSecVertexs = cms.EDProducer("CATSecVertexProducer",
-    muonSrc = cms.InputTag("selectedPatMuonsPFlow"),
-    elecSrc = cms.InputTag("selectedPatElectronsPFlow"),
-    vertexLabel = cms.InputTag("offlineSlimmedPrimaryVertices"),
+    muonSrc = cms.InputTag("slimmedMuons"),
+    elecSrc = cms.InputTag("slimmedElectrons"),
+    vertexLabel = cms.InputTag("catVertex"),
```

Comment:
Was this producer run on MiniAOD? I remember it crashes because of absence of trackExtra in MiniAOD and was not included in the main catTuple production. @geonmo , was the problem already fixed?